tailwind.config.ts

```diff
@@ -1,5 +1,15 @@
 import type { Config } from 'tailwindcss';
 
+const breakpoints = new Map([
+    ['xxs', 320],
+    ['xs', 375],
+    ['sm', 576],
+    ['md', 768],
+    ['lg', 992],
+    ['xl', 1200],
+    ['xxl', 1400],
```

Comment:
Why do these need to be changed?

Reply:
I found that values I've set at first do not match breakpoints from dashh-kit. That's why I've adjusted values to make xss, xs, md, lg literals to match